visualization/dashboard/src/MLIDashboard/Controls/CohortEditor/CohortEditor.styles.ts

```diff
@@ -48,9 +49,10 @@ export const cohortEditorStyles: () => IProcessedStyleSet<ICohortEditorStyles> =
     const theme = getTheme();
     return mergeStyleSets<ICohortEditorStyles>({
         cohortEditor: {
-            overflowY: 'visible',
+            overflowY: 'auto',
             width: '560px',
-            height: '624px',
+            height: '592px',
```

Comment:
I'll double check this works for editing a cohort, for some reason, the height requirements of the dialog seems diffrent when editing a cohort than just adding a cohort....